osbuild/testutil/__init__.py

```diff
@@ -62,6 +63,19 @@ def finder(s): return expected_err in s  # pylint: disable=C0321
                for err_msg in err_msgs), f"{expected_err} not found in {err_msgs}"
 
 
+class MockCommandCallArgs:
```

Comment:
This could use a bit of documentation about what it expects the format of callog to be, and how it can be used.

Reply:
Thank you! That is an excellent point, I will add some docs.